both/access_control.js

```diff
@@ -2,13 +2,16 @@ getCannotManageCompetitionReason = function(userId, competitionId) {
   if(!userId) {
     return new Meteor.Error(401, "Must log in");
   }
+  var user = Meteor.users.findOne(userId, { fields: { siteAdmin: 1 } });
+  if(!user) {
+    return new Meteor.Error(401, "User not found");
```

Comment:
This kind of message can help when things go wrong:

"User '" + userId +"' not found"